### PR TITLE
docs: customize the CSS and graphics

### PR DESCRIPTION
**Pull request by ahoymrag**
### What I did

fixes: #

### How I did it
Updated the CSS file a bit - for the docs 
- Changed the background colors
- Changed the Font Family
- Basic Padding and Leading
### Checklist

- [ ] Passes all linting checks (pre-commit and CI jobs)
- [ ] New test cases have been added and are passing
- [ ] Documentation has been updated
- [ ] PR title follows [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) standard (will be automatically included in the changelog)

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -77,7 +82,7 @@ def fixpath(p):
     get away with having only one copy of the CSS and web fonts.
     """
     new = f"/{project}/latest/_static/" + p.split("_static")[1]
-    return new
+    return p
```
Comment (unparalleled-js) [inline docs/conf.py:85]: Is the variable `new` still needed then? What is the nature of this change?

File: docs/_static/custom.css [hunk 2]
```diff
@@ -1 +1,124 @@
...
+    min-height: 100%;
+    color: #9b9b9b;
+    background: #343131;
+    z-index: 200;
+}
+
+footer {
+    color: grey;
+    display: none;
+}
```
Comment (unparalleled-js) [inline docs/_static/custom.css:124]: Can you add a newline at the end of this file?

File: docs/conf.py [hunk 3]
```diff
@@ -58,6 +58,11 @@ def setup(sphinx):
     sphinx.connect("autoapi-skip-member", autoapi_skip_members)
 
 
+# -- AG CUSTOM CODE BABY
+def setup(app):
+    app.add_css_file('custom.css')
```
Comment (unparalleled-js) [inline docs/conf.py:63]: nit pick, all other strings seems to use double quotes in this file

File: docs/conf.py [hunk 4]
```diff
@@ -58,6 +58,11 @@ def setup(sphinx):
     sphinx.connect("autoapi-skip-member", autoapi_skip_members)
 
 
+# -- AG CUSTOM CODE BABY
```
Comment (unparalleled-js) [inline docs/conf.py:61]: What is AG?

File: docs/_static/custom.css [hunk 5]
```diff
@@ -1 +1,124 @@
-#version-picker-label { color: white; display: inline; }
+@import url('https://fonts.googleapis.com/css?family=Space+Mono&display=swap');
```
Comment (unparalleled-js) [inline docs/_static/custom.css:1]: Can you format this file?
There are lots of unneeded empty lines and some of the indentation is off. There is probably a tool like `prettier` that can help!

**Comment by unparalleled-js** [comment]
Wow this looks awesome!
I am not 100% it is loading correctly for me, can I see a screenshot so I know? Otherwise the rest of my feedback may not make complete sense:

* the font under `User API Reference` seems small. It seems smaller than other fonts like it.
* the font for `Table of Contents` seems wrong and unusual, lime times new roman or something
* the color of the links might be a tad bright for being on a white background, that orange-ish color. I love the color but it is just a tad bright, maybe?
* The user api reference guides are very white, I like maybe the main headers of the guides should be black like they are for the quickstart guides

File: docs/conf.py (hunk 1 above)
Comment (fubuloubu) [reply docs/conf.py:85]: This section needs work. This was a hack to get it working for, but we should figure out exactly what's going on here and why it doesn't work before merging

File: docs/conf.py (hunk 4 above)
Comment (fubuloubu) [reply docs/conf.py:61]: @ahoymrag's initials lol

File: docs/conf.py [hunk 6]
```diff
@@ -58,6 +58,11 @@ def setup(sphinx):
...
+def setup(app):
```
Comment (fubuloubu) [inline docs/conf.py:62]: Type checker is yelling about this function name shadowing an imported function. Should rename this

**Review by fubuloubu — COMMENTED** [review]
Run `black docs/conf.py` and amend previous commit to fix linting bug

File: docs/conf.py (hunk 4 above)
Comment (unparalleled-js) [reply docs/conf.py:61]: immortalized!

**Comment by unparalleled-js** [comment]
Closing as this these changes made it from #302 ! Thank you so much @ahoymrag 
It should be much easier to contribute to the docs now.

**Pull request closed without merging**